src/wp-admin/includes/template.php

```diff
@@ -2108,6 +2108,10 @@ function _admin_search_query() {
  * @param bool   $deprecated Not used.
  */
 function iframe_header( $title = '', $deprecated = false ) {
+	if ( false !== $deprecated ) {
+		_deprecated_argument( __FUNCTION__, '2.7.0' );
```

Comment:
```suggestion
		_deprecated_argument( __FUNCTION__, '4.2.0' );
```
Deprecated in https://core.trac.wordpress.org/ticket/31309.